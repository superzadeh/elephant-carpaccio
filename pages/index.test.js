import React from 'react'
import renderer from 'react-test-renderer'

import Index from './index'

const defaultProps = {
}
const getElement = (propOverrides = {}) => {
  const props = Object.assign({}, defaultProps, propOverrides)
  const element = renderer.create(<Index {...props} />).toJSON()
  return { props, element }
}

describe('index', () => {
  it('should render correctly', () => {
    const { element } = getElement()
    expect(element).toMatchSnapshot()
  })
})
