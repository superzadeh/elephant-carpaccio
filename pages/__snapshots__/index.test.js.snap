// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should render correctly 1`] = `
<div
  className="container"
>
  <h1>
    Elephant Carpaccio
  </h1>
  <br />
  <label
    htmlFor="quantity"
  >
    Number of Items
  </label>
   
  <input
    name="quantity"
    onChange={[Function]}
    type="text"
  />
  <br />
  <label
    htmlFor="price"
  >
    Price per Item
  </label>
   
  <input
    name="price"
    onChange={[Function]}
    type="text"
  />
  <br />
  <label
    htmlFor="stateCode"
  >
    State
  </label>
   
  <input
    name="stateCode"
    onChange={[Function]}
    type="text"
  />
  <br />
  <br />
  <div>
    Total price without discount: 
    0
    $
  </div>
  <div>
    Total price with discount: 
    <strong>
      0
      $
    </strong>
  </div>
  <div>
    You just saved 
    $
  </div>
  <div>
    Total amount to pay (tax included) 
    $
  </div>
  <style
    global={true}
    jsx={true}
  >
    
* {
  color: #222;
  font-weight: 200;
  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
  margin: 0;
  box-sizing: border-box;
}

body,
p {
  font-size: 16px;
  line-height: 24px;
}

h1 {
  font-size: 40;
  font-weight: 200;
  line-height: 40px;
}

h2,
h3,
h4,
h5,
h6 {
  font-weight: 500;
  margin-top: 20px;
}

h2,
h3,
h4 {
  margin-bottom: 15px;
}

h2 a,
h3 a,
h4 a {
  font-weight: 500;
  text-decoration: none;
}

h2 a::before,
h3 a::before,
h4 a::before {
  content: '';
  display: block;
  height: 70px; /* fixed header height*/
  margin: -70px 0 0; /* negative fixed header height */
}

strong {
  font-weight: bolder;
}

em {
  font-style: italic;
}

ul {
  margin: 0;
}
ul.dashed {
  list-style-type: none;
}
ul.dashed &gt; li {
  text-indent: -5px;
}
ul.dashed &gt; li:before {
  content: '- ';
  text-indent: -5px;
}

/* Main content */
.main {
  margin-top: 70px;
}

/* Quotes */
blockquote {
  margin: 20px 50px;
}

.headline {
  text-decoration: underline;
}

/* Tables */
th,
td {
  margin: 0;
  padding: 5px;
  vertical-align: top;
}
td:nth-child(1) {
  width: 48%;
}
td ul {
  padding-left: 15px;
}
td blockquote {
  margin: 10px 10px 10px 30px;
}
th {
  text-align: center;
  border-bottom: 1px solid #333;
}
td {
  border-bottom: 1px solid #333;
}
td:nth-child(2) {
  border-left: 1px solid #333;
}

  </style>
</div>
`;
